dandiapi/api/tests/test_asset.py

```diff
@@ -208,6 +209,11 @@ def test_asset_rest_retrieve(api_client, version, asset):
         == asset.metadata.metadata
     )
 
+    assert (
+        json.loads(api_client.get(f'/api/assets/{asset.asset_id}/metadata/').content)
+        == asset.metadata.metadata
+    )
+
```

Comment:
I would put this in a new test, `test_asset_direct_metadata` and put it at the bottom of the file. So far the pattern for these API tests is to test one endpoint per test.

Reply:
See 7f46190.